src/hooks/verifyPluginVersion.ts

```diff
@@ -9,7 +9,12 @@ import { Hook } from '@oclif/config';
 import { compareVersions, isVersion } from '../versions';
 
 const FORCE_PLUGINS = ['salesforce-alm', 'force-language-services'];
-
+const BANNED_PLUGINS: Record<string, string> = {
+  salesforcedx: `The salesforcedx plugin is deprecated.
+    Installing it manually via 'sfdx plugins:install salesforcedx' is no longer supported and can result in duplicate commands and outdated plugins.
+    See https://github.com/forcedotcom/cli/issues/1016 for more information about this change.`,
+  'sfdx-cli': 'Installing sfdx-cli is not allowed.',
```

Comment:
Let's add a more descriptive message here to point users in the right direction to install older versions of the cli. Something like: `'sfdx-cli' cannot be installed as a plugin. If you are trying to install an older version of the cli, visit https://sfdc.co/install-older-cli-versions for instructions`

(I just created that Shrink (https://www.sfdc.co/) short link, it may take a few minutes to work)

Reply:
Logic looks good 👍